shardingsphere-infra/shardingsphere-infra-executor/src/test/resources/sql/federate_data_0.sql

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+DELETE FROM t_order_federate;
+DELETE FROM t_user_info;
+DELETE FROM t_order_federate_sharding_0;
+DELETE FROM t_order_federate_sharding_1;
+INSERT INTO t_order_federate VALUES(1000, 0, 'init');
+INSERT INTO t_order_federate VALUES(1001, 1, 'init');
+
```

Comment:
Could we remove this useless line?
Once this line is removed, I suppose it is time to merge. BTW, could we add more cases to cover more optimization cases?

Reply:
@tristaZero OK. And I've modified this file.